commons-rest/src/main/java/com/mule/connectors/commons/rest/builder/RequestBuilder.java

```diff
@@ -6,6 +6,7 @@
 import com.mule.connectors.commons.rest.builder.handler.ResponseHandler;
 import com.mule.connectors.commons.rest.builder.strategy.*;
 import org.apache.commons.lang3.StringUtils;
+import org.glassfish.jersey.internal.util.Base64;
```

Comment:
We may use this:
https://commons.apache.org/proper/commons-codec/apidocs/org/apache/commons/codec/binary/Base64.html